### ISSUE TITLE
Cannot find module '@angular-devkit/core'

### ISSUE THREAD

**Issue by adrianwix** [nx-go/nx-go#44]
When trying to generate the API I get "Cannot find module '@angular-devkit/core'"
```
Require stack:
- C:\Users\Adrian Wix\Code\agency\go-playground\node_modules\@nrwl\tao\src\commands\ngcli-adapter.js
- C:\Users\Adrian Wix\Code\agency\go-playground\node_modules\@nrwl\tao\src\commands\generate.js
- C:\Users\Adrian Wix\Code\agency\go-playground\node_modules\@nrwl\tao\index.js
- C:\Users\Adrian Wix\Code\agency\go-playground\node_modules\@nrwl\cli\lib\init-local.js
- C:\Users\Adrian Wix\Code\agency\go-playground\node_modules\@nrwl\cli\bin\nx.js
- C:\Users\Adrian Wix\AppData\Roaming\npm\node_modules\nx\node_modules\@nrwl\cli\bin\nx.js
- C:\Users\Adrian Wix\AppData\Roaming\npm\node_modules\nx\bin\nx.js
```

**Comment by ghazninattarshah** [comment]
I've got the same error, but here are the steps.

1. created a workspace
```
npx create-nx-workspace --preset=react
```
2. installed nx-go
```
yarn add -D @nx-go/nx-go
```
3. Tried to generate go app
```
nx g @nx-go/nx-go:app api1
```

Expected: it should've generated the go project
Actual: Received an error that's part of description `"Cannot find module '@angular-devkit/core'"`

**Comment by ghazninattarshah** [comment]
The workaround is to manually install below packages.

```
yarn add -D @angular-devkit/core
yarn add -D @angular-devkit/schematics
```

**Comment by akolybelnikov** [comment]
I'm getting `NOT SUPPORTED: keyword "id", use "$id" for schema ID` after I have installed the missing @angular packages and tried generating an app again.

**Comment by tonizt** [comment]
> I'm getting `NOT SUPPORTED: keyword "id", use "$id" for schema ID` after I have installed the missing @angular packages and tried generating an app again.

i was able to "fix" this by editing the file `node_modules/@nx-go/nx-go/src/schematics/application/schema.json` and changing `id` to `$id`. After that re run the plugin, it will work.
Also, when running the `nx build <app-name>` command, you'll get similar error 'Cannot find module @angular-devkit/architect'. Install that dependency to make it work.
For now all of the scripts are working, i'm just getting this warning that I hope doesn't cause issues in the future:

```
[NX] Angular devkit called `writeWorkspace`, this may have had unintended consequences in workspace.json
[NX] Double check workspace.json before proceeding
CREATE apps/api/main.go
CREATE apps/api/main_test.go
UPDATE workspace.json
```

**Comment by beeman** [comment]
I just released [@nx-go/nx-go v2.0.0](https://www.npmjs.com/package/@nx-go/nx-go/v/2.0.0) which should fix these issues. 

Please give it a try and if it doesn't fix it please provide a reproduction.
